README.md

```diff
@@ -240,52 +141,39 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/henask12/To-do-list/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
+If you like this project and find it helpful, please consider showing your support. Here are a few ways you can contribute:
 
-If you like this project...
+-   Star this repository on GitHub.
+-   Share the project with others who might find it useful.
+-   Contribute to the project by submitting bug reports, feature requests, or pull requests.
+
+Your support and feedback are greatly appreciated!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+I would like to thank the following individuals and resources for their inspiration:
 
-I would like to thank...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+-   My codding Partner's
+-   Microverse Communtiy
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./MIT) licensed.
```

Comment:
- [ ] You are yet to fix the MIT link. When I click it I should be taken to the MIT.md file but instead I am led to this. Kindly fix that.

  ![image](https://github.com/henask12/To-do-list/assets/36542986/98be179a-f0d9-4467-b05e-82c9ece62932)

Reply:
@TedLivist you are checking the master branch 
on the todolist branch it is already fixed